bundles/org.openhab.binding.paradoxalarm/src/main/java/org/openhab/binding/paradoxalarm/internal/communication/EvoCommunicator.java

```diff
@@ -217,7 +217,7 @@ private void createZoneTamperedFlags(ZoneStateFlags result, byte[] firstPage, by
     private void createZoneLowbatteryFlags(ZoneStateFlags result, byte[] firstPage, byte[] secondPage) {
         int pageOffset = panelType == PanelType.EVO48 ? 58 : 64;
         byte[] firstBlock = Arrays.copyOfRange(firstPage, 52, pageOffset);
-        if (panelType != PanelType.EVO192) {
+        if (PanelType.isBigRamEvo(panelType)) {
```

Comment:
Is it correct, or did you mean:
```suggestion
        if (!PanelType.isBigRamEvo(panelType)) {
```

like previously and like the other similar changes?

Reply:
Woot...., definitely overlooked. Wonder how it works correctly on my system :P